packages/nextjs/src/utils/config.ts

```diff
@@ -30,32 +30,50 @@ type EntryPointObject = { import: string | Array<string> };
 const _injectFile = (entryProperty: EntryPropertyObject, injectionPoint: string, injectee: string): void => {
   // can be a string, array of strings, or object whose `import` property is one of those two
   let injectedInto = entryProperty[injectionPoint];
-  // whatever the format, add in the sentry file
-  injectedInto =
-    typeof injectedInto === 'string'
-      ? // string case
-        [injectee, injectedInto]
-      : // not a string, must be an array or object
-      Array.isArray(injectedInto)
-      ? // array case
-        [injectee, ...injectedInto]
-      : // object case
-        {
-          ...injectedInto,
-          import:
-            typeof injectedInto.import === 'string'
-              ? // string case for inner property
-                [injectee, injectedInto.import]
-              : // array case for inner property
-                [injectee, ...injectedInto.import],
-        };
+
+  // Sometimes especially for older next.js versions it happens we don't have an entry point
+  if (injectedInto === undefined) {
```

Comment:
Not using a strict check will also check for `null` and `''`, besides avoiding checking for `null` again on line 46:

```suggestion
  if (!injectedInto) {
```